flang/lib/Lower/Bridge.cpp

```diff
@@ -1979,8 +1979,8 @@ class FirConverter : public Fortran::lower::AbstractConverter {
     const auto isHostAssoc =
         Fortran::semantics::IsHostAssociated(sym, sym.owner());
     Fortran::lower::CharacterExprHelper charHelp{*builder, loc};
-    Fortran::lower::BoxAnalyzer sba;
-    sba.analyze(sym);
+    Fortran::lower::BoxAnalyzer ba;
+    ba.analyze(sym);
```

Comment:
Don't change this variable name. Thanks.

Reply:
sba is no longer a reasonable choice.  Do you have a better suggestion?

Reply:
Not really asking.

Reply:
Ok, but please clean this up.